src/commands/destroy/datacenter.ts

```diff
@@ -41,12 +40,7 @@ async function destroy_datacenter_action(options: DestroyDatacenterOptions, name
     }
   }
 
-  const confirm = options.autoApprove || (await Inputs.promptForContinuation('Are you sure you want to proceed?'));
-
-  if (!confirm) {
-    console.error('Datacenter destruction cancelled');
-    Deno.exit(1);
-  }
+  await command_helper.pipelineRenderer.confirmPipeline(pipeline, options.autoApprove);
```

Comment:
Note to self: This change may actually be slightly incorrect. When deleting a datacenter it wasn't showing the pipeline preview before asking you to confirm, but I think this breaks the case where there's environments to still delete so I need to double check.